src/coreclr/src/debug/daccess/daccess.cpp

```diff
@@ -72,11 +72,11 @@ BOOL WINAPI DllMain(HANDLE instance, DWORD reason, LPVOID reserved)
         {
             return FALSE;
         }
-#endif
-        InitializeCriticalSection(&g_dacCritSec);
-
+#else
         // Save the module handle.
         g_thisModule = (HINSTANCE)instance;
```

Comment:
Is used for anything? Now that it is Windows-only, can the uses be replaced by `__ImageBase` if there are any left?

Reply:
I was less sure about uses in debugger related stuff. I am not familiar with that enough to know for sure is windows-only, will not be used indirectly by SOS and similar, so I figured it might be safer to leave debugger use as it was.

Reply:
`g_thisModule` does not have any of the DAC macros. I cannot by used indirectly by anything. If you delete `g_thisModule` and `GetModuleInst` from this file, is it going to break anything?

Reply:
I will try. I assume we can trust tests here?    (I had some cases in the past breaking SOS without any tests failing)

Reply:
This ends up exposed via `GetDacModuleHandle`/`ISOSDacInterface_GetDacModuleHandle`.
If that is Windows only we could just use __ImageBase.

Reply:
> GetDacModuleHandle

Ugh. Then we should either keep `g_thisModule` for both Windows and Unix. Or delete it and change `GetDacModuleHandle` to return `__ImageBase` on Windows and fail on Unix.

Reply:
>  return __ImageBase on Windows and fail on Unix

Did that.